joomla/joomla-cms
# [5.0] replace deprecated Factory::getLanguage() code

### Summary of Changes
Following the dictat that core should not be using any deprecated code this pr replaces all remaining uses of Factory::getLanguage() with its replacement Factory::getApplication()->getLanguage()

### Testing Instructions
1. enable debug lang and navigate the site and check that the languages load and the debug plugin reports no errors
2. change `<rtl>0</rtl>` to `<rtl>1<rtl>` in langmetadata.xml and repeat


### Actual result BEFORE applying this Pull Request
deprecated code is used


### Expected result AFTER applying this Pull Request
no deprecated code and everything still works as expected


### Link to documentations
Please select:

i assume no documentation is needed for this
